chart/kubeapps/README.md

```diff
@@ -324,8 +324,11 @@ It is possible that when upgrading Kubeapps an error appears. That can be caused
 
 > Note: These steps assume that you have installed Kubeapps in the namespace `kubeapps` using the name `kubeapps`. If that is not the case replace the command with your namespace and/or name.
 
+> Note: If you are upgrading from 2.3.1 see the [following section](#upgrading-to-2-3-1).
+
 > Note: If you are upgrading from 1.X to 2.X see the [following section](#upgrading-to-2-0).
```

Comment:
Do you think it's safe to remove the `#upgrading-to-2-0` related content now? Hmm, guess it's only been 5 months.